lib/src/cli.rs

```diff
@@ -46,6 +46,11 @@ pub(crate) struct UpgradeOpts {
     /// a userspace-only restart.
     #[clap(long, conflicts_with = "check")]
     pub(crate) apply: bool,
+
+    /// Skip TLS and certificate verification.
+    /// This is very insecure and should only be used in testing environments
+    #[clap(long)]
+    pub(crate) insecure_disable_tls_verification: bool,
```

Comment:
I think this should always be part of the source spec; unlike `podman` I'm trying here to be declarative where we can.

This means that once you do a `bootc switch --insecure-disable-tls-verification`, it is "sticky" and applied automatically on subsequent `bootc upgrade`s.

So we can just drop this hunk I believe.